src/routers/workspaces.py

```
@@ -114,30 +114,23 @@ async def search_workspace(
 )
 async def get_deriver_status(
     workspace_id: str = Path(..., description="ID of the workspace"),
-    peer_id: str | None = Query(None, description="Optional peer ID to filter by"),
+    observer_id: str | None = Query(
+        None, description="Optional observer ID to filter by"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)